orcid-core/src/main/java/org/orcid/core/adapter/jsonidentifier/converter/JSONWorkExternalIdentifiersConverterV3.java

```diff
@@ -86,8 +90,17 @@ public ExternalIDs convertFrom(String source, Type<ExternalIDs> destinationType)
                     if (urlValidator.isValid(workExternalIdentifier.getUrl().getValue())){
                         id.setNormalizedUrl(new TransientNonEmptyString(workExternalIdentifier.getUrl().getValue()));
                     }                    
+                } else {
+                    try {
+                        PIDResolutionResult pdr = resolverService.resolve(id.getType(), workExternalIdentifier.getWorkExternalIdentifierId().content); 
+                        if (pdr.isResolved()) {
+                            id.setNormalizedUrl(new TransientNonEmptyString(pdr.getGeneratedUrl()));
+                        }                       
+                    } catch (IllegalArgumentException iae) {
+
+                    }
```

Comment:
@DanielPalafox looks like this will change the API behavior, it will add the normalized url to the response which is a change in which the API works; could you please check that in the member API and let me know the result?